styles/layout.css

```diff
@@ -0,0 +1,121 @@
+@media(max-width: 768px) {
```

Comment:
you have your CSS organized incorrectly. You design for your mobile first view without a media query rule. then any properties you want differently for non-mobile-view you would nest under a @media. So you should have just have one media query. I would review the readings on this.